src/util/anchor.ts

```diff
@@ -170,3 +170,31 @@ export async function getEpochInfo(
     slotsRemainingInEpoch,
   }
 }
+
+export async function createAccount({
```

Comment:
Why the wrapper around the create account? I would use directly the SystemProgram.createAccount in marinade.ts

Reply:
I updated the code for not creating a helper method.